index.html

```diff
@@ -21,7 +21,7 @@
     - The freeCodeCamp Team
   -->
 
-  <p class="paragraph">Hello World, we are software developers!</p>
+  <p class="paragraph">Hello World, we are software developers!!</p>
 
```

Comment:
Please use a semantic header tag to wrap up the `p` tag, so it can be more understandable especially to screen readers.